vng_api_common/authorizations/models.py

```diff
@@ -1,27 +1,62 @@
 import uuid
+from typing import Optional
 
 from django.contrib.postgres.fields import ArrayField
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
+from solo.models import SingletonModel
+from zgw_consumers.constants import APITypes, AuthTypes
+
+from vng_api_common.client import Client, get_client
+
 from ..constants import ComponentTypes, VertrouwelijkheidsAanduiding
 from ..decorators import field_default
 from ..fields import VertrouwelijkheidsAanduidingField
-from ..models import APIMixin, ClientConfig
+from ..models import APIMixin
+
 
+class AuthorizationsConfigManager(models.Manager):
+    def get_queryset(self):
+        qs = super().get_queryset()
+        return qs.select_related("authorizations_api_service")
 
-@field_default("api_root", "https://autorisaties-api.vng.cloud/api/v1")
-class AuthorizationsConfig(ClientConfig):
+
+class AuthorizationsConfig(SingletonModel):
     component = models.CharField(
         _("component"),
         max_length=50,
         choices=ComponentTypes.choices,
         default=ComponentTypes.zrc,
     )
 
+    authorizations_api_service = models.ForeignKey(
+        "zgw_consumers.Service",
+        limit_choices_to=dict(
+            api_type=APITypes.ac,
+            auth_type=AuthTypes.zgw,
+        ),
+        verbose_name=_("autorisations api service"),
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
```

Comment:
I do think it should not be nullable.

Reply:
I've added a migration to remove null=True